requirements/frozen.py3.txt

```diff
@@ -305,7 +305,7 @@ Werkzeug==0.14.1
 widgetastic.core==0.30.2
 widgetastic.patternfly==0.0.39
 widgetsnbextension==3.4.2
-wrapanapi==3.0.30
+wrapanapi==3.0.32
```

Comment:
3.0.33 is available and includes a recently merged refactor of some of the RedFish objects.

Reply:
👍